checks.d/docker_daemon.py

```diff
@@ -818,7 +835,13 @@ def _crawl_container_pids(self, container_dict):
                         continue
                     container_dict[container_id]['_pid'] = folder
                     container_dict[container_id]['_proc_root'] = os.path.join(proc_path, folder)
-            except Exception as e:
+                else: # if we match by pid that should be enough (?) - O(n) ugh!
```

Comment:
should we use something like `elif :` here? I'm just trying to see if we can avoid entering this loop if it's not absolutely necessary.